WebService/ComparisonEngineUI/ComparisonEngineUI/AppShell.xaml.cs

```diff
@@ -19,6 +19,7 @@ public AppShell()
             Routing.RegisterRoute(nameof(EditDrinkPricePage), typeof(EditDrinkPricePage));
             Routing.RegisterRoute(nameof(AddBarPage), typeof(AddBarPage));
             Routing.RegisterRoute(nameof(AddDrinkPage), typeof(AddDrinkPage));
+            Routing.RegisterRoute(nameof(BarPage), typeof(BarPage));
         }
```

Comment:
shouldn't there be a Routing.RegisterRoute(nameof(DrinkPage), typeof(DrinkPage)); as well?

Reply:
Fairly certain this is left over from how I first tried to make those pages work. It should work without registering the Route for BarPage